src/Io/KeyboardActionMapping.cpp

```diff
@@ -320,44 +320,26 @@ GameConfig::String *KeyboardActionMapping::InputActionToConfigKey(InputAction ac
 }
 
 PlatformKey KeyboardActionMapping::ConfigDefaultKey(InputAction action) {
-    PlatformKey key = PlatformKey::None;
-    ConfigValue<std::string> *val = InputActionToConfigKey(action);
-
-    if (val)
-        TryParseDisplayName(val->Default(), &key);
-
-    return key;
+    ConfigValue<PlatformKey> *val = InputActionToConfigKey(action);
+    return val ? val->Default() : PlatformKey::None;
 }
 
 PlatformKey KeyboardActionMapping::ConfigGetKey(InputAction action) {
-    PlatformKey key = PlatformKey::None;
-    ConfigValue<std::string> *val = InputActionToConfigKey(action);
-
-    if (val)
-        TryParseDisplayName(val->Get(), &key);
-
-    return key;
+    ConfigValue<PlatformKey> *val = InputActionToConfigKey(action);
+    return val ? val->Default() : PlatformKey::None;
```

Comment:
return val ? val->Get() : PlatformKey::None;

Reply:
Missed it, thanks!